idea_town/frontend/static-src/app/templates/experiment-page.js

```diff
@@ -22,55 +22,59 @@ export default `
       <div class="details-content">
         <div class="details-overview">
           <img src="{{thumbnail}}" width="260">
-          <section>
+          {{#model.measurements}}
+          <section class="measurement">
             <h3>Measurements</h3>
-            <p class="disclaimer">All data is collected anonymously and used only to help us improve this test. <a href="">Learn more.</a></p>
+            <p class="disclaimer">All data is collected anonymously and used only to help us improve this test.</p>
+            {{{model.measurements}}}
+            <!--
             <ul class="measurement">
               <li>Usage per session</li>
               <li>Index of selected results</li>
               <li>Time spent searching</li>
             </ul>
+            -->
           </section>
+          {{/model.measurements}}
           <section>
             <h3>Brought to you by</h3>
             <ul class="contributors">
+              {{#model.contributors}}
               <li>
-                <img src="images/default-avatar@2x.png" width="56" height="56">
-                <div class="contributor">
-                  <p class="name">Chris P. Bacon</span>
-                  <p class="title">Senior Strategy Dingus</span>
-                </div>
-              </li>
-              <li>
-                <img src="images/default-avatar@2x.png" width="56" height="56">
-                <div class="contributor">
-                  <p class="name">Chris P. Bacon</span>
-                  <p class="title">Senior Strategy Dingus</span>
-                </div>
-              </li>
-              <li>
-                <img src="images/default-avatar@2x.png" width="56" height="56">
+                <img src="{{avatar}}" width="56" height="56">
                 <div class="contributor">
-                  <p class="name">Chris P. Bacon</span>
-                  <p class="title">Senior Strategy Dingus</span>
+                  <p class="name">{{display_name}}</span>
```

Comment:
the other templates use lowerCamelCase for names, seems like we probably want to keep snake_case on the python side